tools/repoctl/internal/repoctl/affected.go

```
@@ -0,0 +1,92 @@
+	sort.Slice(out, func(i, j int) bool { return assetNum(out[i]) < assetNum(out[j]) })
+	return out
+}
+
+func assetNum(t string) int {
+	n, _ := strconv.Atoi(strings.TrimPrefix(t, "T"))
+	return n
+}
+
+// Affected 跑 `git -C root diff --name-only base` 并映射资产；git 失败 → 错误（exit 2）。
+func Affected(root, base string) ([]string, error) {
+	out, err := exec.Command("git", "-C", root, "diff", "--name-only", base).Output()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)